docs/user/AggFunc.md

```diff
@@ -0,0 +1,112 @@
+# Aggregate Function
+Aggregate functions output a single result value from a collection of input values.
+
+Unlike SQL, `GROUP BY` is not required to use the aggregate function. 
+
+Unless otherwise specified, aggregate functions will skip `NULL` and `MISSING`
+
+## AVG
+
+Returns the average (arithmetic mean) of the input expression values.
+
+Signature
+: `AVG: COLLECTION OF NUMERIC TYPE -> DECIMAL OR NULL`
+
+Header
+: `AVG(expr)`
+
+Purpose
+: Given a collection of numeric type values, evaluates the arithmetic mean of the input expression values. 
```

Comment:
I think this can be consolidated to just **Description** and **Signature**. The description and purpose are the same, and the HEADER doesn't tell us any more than the signature. Signature should be plural.

Consider,

## AVG

### Description
Given a collection of numeric type values, `AVG` evaluates the arithmetic mean of the input expression values.

### Signatures
`AVG(collection<numeric>) -> decimal or NULL`

Reply:
Addressed in the new commit.